pages/server/software/gitea.mdx

```diff
@@ -11,17 +21,120 @@ Apart from a Git with a cup of Tea:
 
 ---
 
-We ask you to read entirely this document because if you skip one step you might break the server and you will need to reinstall everything.
+It is important that you thoroughly read and follow this document, as one mistake might end up in you having to re-make the server.
 
 # Creating the server
 
-To create a server, we suppose that you've already [created and linked an account into the panel.](/getting-started)
```

Comment:
This really isn’t needed to be change, the period can stay there.

Reply:
oh i changed the entire code by copying and pasting them so theyre all similiar
i can add it back if you want